### ISSUE TITLE
Add a command line tool

### ISSUE THREAD

**Issue by erikvold** [saadtazi/firefox-profile-js#51]
It would be nice if I were able to create a fx profile like so with terminal `firefox-profile ./temp/`

**Comment by saadtazi** [comment]
Nice idea. What do you think of:

* `firefox-profile`: create a blank profile in system tmp folder
*  `firefox-profile ./mytmp`: create a blank profile in `./mytmp`
* `firefox-profile -c myprofile [outputfolder]`: copy an existing profile

Any other use cases you can think of?

**Comment by saadtazi** [comment]
I create a `add-cli` branch with a cli:

```
usage: firefox-profile [-v] [-p profilename || -c profile_dir] [-e extension_path1 [-e extension_path2]] [-o destination_dir || --ouput destination_dir || -b [file_path] || --base64 [file_path]]

-v: verbose mode (same as --verbose)
-h: show this message (same as --help)
-p profilename: profile to copy (same as --profile)
-c profile_dir: profile folder path to copy (same as --copy)
-e extension_path: file path to extension to add to the profile. Can be present multiple times (same as --extension)
-o destination_dir (or --output destination_dir): folder where the profile will be created.
-b [file_path](or --base64): write the encoded firefox profile to stdout or to file_path if specified.
```

I published it as v0.3.11. Let me know if it fits your need.

Note that it does not support user prefs yet.

**Comment by erikvold** [comment]
Nice!
